src/commands/case_management.py

```diff
@@ -477,15 +475,19 @@ async def cmd_case_management_quiet(ctx: Context):
         await ctx.reply("Got no information yet")
         return
 
-    delta = relativedelta(datetime.now(tz=timezone.utc), ctx.bot.board.last_case_datetime)
-    hour_part = ""
-    if delta.hours:
-        hour_plural = 's' if delta.hours > 1 else ''
-        hour_part = f"{delta.hours} hour{hour_plural} and "
-    minute_plural = 's' if delta.minutes != 1 else ''
-    await ctx.reply(
-        f"The last case was created {hour_part}{delta.minutes} minute{minute_plural} ago."
-    )
+    # delta = pendulum.now(tz=pendulum.tz.UTC) - ctx.bot.board.last_case_datetime
+    # hour_part = ""
+    # if delta.hours:
+    #     hour_plural = 's' if delta.hours > 1 else ''
+    #     hour_part = f"{delta.hours} hour{hour_plural} and "
+    # minute_plural = 's' if delta.minutes != 1 else ''
+
```

Comment:
[error] purge the old implementation. If its needed later it can be retrieved from VCS history.